package.json

```diff
@@ -18,13 +18,13 @@
     "typings"
   ],
   "scripts": {
-    "build": "truffle compile --all && yarn gen:contract-typings && tsc -b",
+    "build": "yarn compile && yarn gen:typings && tsc -b",
```

Comment:
Maybe double check that the bytecode doesn't change, because if it did we should do a major version bump.